docs/shared/native-plugin-notice.mdx

```diff
@@ -1,8 +1,12 @@
+<Note>
 ⚠️ Apollo doesn't recommend creating native plugins for the Apollo Router Core or GraphOS Router, for the following reasons:
 
 - Native plugins require familiarity with programming in Rust.
 - Native plugins require compiling a custom router binary from source, which can introduce unexpected behavior in your router that's difficult to diagnose and support.
+- Custom router binaries are not covered by GraphOS commercial support.
```

Comment:
@smyrick what do we mean by this exactly? Does this mean we don't / can't answer questions about any custom binaries?